backend/src/main/scala/cromwell/backend/io/JobPaths.scala

```diff
@@ -32,6 +32,7 @@ trait JobPaths {
 
   def workflowPaths: WorkflowPaths
   def returnCodeFilename: String = "rc"
+  def doubleMemoryRetryRCFilename: String = "double_memory_retry_rc"
```

Comment:
Assuming we remove the hardcoding of the multiplier, how about `val memoryRetryRcFilename = "memory_retry_rc"`?